grafana-plugin/e2e-tests/alerts/sms.test.ts

```diff
@@ -5,8 +5,7 @@ import { createIntegrationAndSendDemoAlert } from '../utils/integrations';
 import { waitForSms } from '../utils/phone';
 import { configureUserNotificationSettings, verifyUserPhoneNumber } from '../utils/userSettings';
 
-// TODO: enable once we've signed up for a MailSlurp account to receieve SMSes
-test.skip('we can verify our phone number + receive an SMS alert', async ({ adminRolePage }) => {
+test('we can verify our phone number + receive an SMS alert @expensive', async ({ adminRolePage }) => {
```

Comment:
`@expensive` annotation here is important. Relevant to the [modified commands in `package.json`](https://github.com/grafana/oncall/pull/2751/files#diff-5c26e441d662e382bbb09a0f0b4e5b18eabf786f1fff0a19500f9c1bffb45699R14-R15)